pysmt/smtlib/script.py

```diff
@@ -392,3 +419,36 @@ def evaluate_command(cmd, solver):
                                   "Please open a bug-report." % cmd.name)
     else:
         raise UnknownSmtLibCommandError(cmd.name)
+
+def omt_evaluate_command(cmd, optimizer, client_data = None):
+
+    if cmd.name == smtcmd.MAXIMIZE:
+        rt = MaximizationGoal(cmd.args[0])
+        if client_data is not None:
+            client_data[0].append(rt)
+        return rt
+    elif cmd.name == smtcmd.MINIMIZE:
+        rt = MinimizationGoal(cmd.args[0])
+        if client_data is not None:
+            client_data[0].append(rt)
+        return rt
+    elif cmd.name == smtcmd.CHECK_SAT:
+        if client_data != None:
```

Comment:
Never use `!=` with `None`
```suggestion
        if client_data is not None:
```